mina-payout-reports/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
**suggestion:** This makes the chart unable to render given that value is not declared.